src/drivers/github.js

```diff
@@ -165,6 +168,15 @@ class Github {
       status = 'completed'
     } = opts;
 
+    const warning =
+      'This command only works inside a Github runner or a Github app.';
+
+    if (!CI || TPI_TASK) winston.warn(warning);
+    if (GITHUB_TOKEN && GITHUB_TOKEN !== this.token)
+      winston.warn(
+        `Your token is different than the GITHUB_TOKEN, this command does not work with PAT. ${warning}`
+      );
```

Comment:
this warning will incorrectly be triggered when using `REPO_TOKEN=<some_app_token>`?

Reply:
Yes, but we cant distinguish between PAT or app token, thats why is just a warning

Reply:
@casperdcl ?

Reply:
something like this?

```suggestion
    if (GITHUB_TOKEN && GITHUB_TOKEN !== this.token && !REPO_TOKEN)
      winston.warn(
        `Your token is different than the GITHUB_TOKEN, this command does not work with PAT. ${warning}`
      );
```

Reply:
No, the warning is just in case you use a PAT instead of a GITHUB_TOKEN

Reply:
The code is OK as it is

Reply:
but what happens if someone has an env with both `REPO_TOKEN` and `GITHUB_TOKEN` defined? this is common afaik...

Reply:
It will warn about bad token usage so they have to explicitly set the `--token $GITHUB_TOKEN` in order to make it work. Im not enforcing it always to read GITHUB_TOKEN because  you could actually use this command with a Github App token